apm-sniffer/apm-sdk-plugin/httpasyncclient-4.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/httpasyncclient/v4/SessionRequestCompleteInterceptor.java

```diff
@@ -34,21 +34,23 @@
 public class SessionRequestCompleteInterceptor implements InstanceMethodsAroundInterceptor {
 
     public static ThreadLocal<HttpContext> CONTEXT_LOCAL = new ThreadLocal<HttpContext>();
+    public static ThreadLocal<Boolean> CONTEXT_LOCAL_NOT_EXIT = new ThreadLocal<Boolean>();
```

Comment:
You didn't call `remove()`, but I think it's necessary. Right?

Reply:
ACK.